subt/ros/proxy/ros_proxy_node.cc

```diff
@@ -104,24 +96,6 @@ void clockCallback(const rosgraph_msgs::Clock::ConstPtr& msg)
   g_countClock++;
 }
 
-void scanCallback(const sensor_msgs::LaserScan::ConstPtr& msg)
-{
-  ros::SerializedMessage sm = ros::serialization::serializeMessage(*msg);
-  protected_zmq_send(g_responder, sm.buf.get(), sm.num_bytes, 0);
-  if(g_countScan % 100 == 0)
-    ROS_INFO("received Scan %d", g_countScan);
-  g_countScan++;
-}
-
-void imageCallback(const sensor_msgs::CompressedImage::ConstPtr& msg)
-{
-  ros::SerializedMessage sm = ros::serialization::serializeMessage(*msg);
-  protected_zmq_send(g_responder, sm.buf.get(), sm.num_bytes, 0);
-  if(g_countImage % 100 == 0)
-    ROS_INFO("received Image %d", g_countImage);
-  g_countImage++;
-}
-
 void odomCallback(const nav_msgs::Odometry::ConstPtr& msg)
```

Comment:
Do you plan to also move odom and gas callbacks over to the python proxy?

Reply:
Well, not really, unless you ask for it. Gas is common for all robots and in `rosmsg` is only "report on change" = very short stream and obvious info, if robot was somewhere with gas.

The odometry is kind of symmetrical to vehicle motion/Twist, which also makes some small mix for 2D and 3D control ... shortly my motivation is low (and I was probably the only one not too much complaining about `rosmsg`).

If we could abandon rosproxy completely then I would do it. (I will probably also modify `rosmsg` in order not to publish empty stream and only support named topics like in Moon project, but this is lower on the priority list ...